src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -2997,5 +2997,35 @@ public static void Main()
 
             Assert.Throws<OperationCanceledException>(() => compilation.Emit(broken));
         }
+
+        [Fact]
+        [WorkItem(11691, "https://github.com/dotnet/roslyn/issues/11691")]
+        public void ObsoleteAttributeOverride()
+        {
+            string source = @"
+using System;
+public abstract class BaseClass<T>
+{
+    public abstract int Method(T input);
+}
+
+public class DerivingClass<T> : BaseClass<T>
+{
+    [Obsolete(""Deprecated"")]
+    public override void Method(T input)
```

Comment:
Have we checked to see if VB has the same bug?  Even if not we should add a regression suite.